lib/presentation/screens/greeting_screen/greeting_layout.dart

```diff
@@ -0,0 +1,36 @@
+import 'package:flutter/material.dart';
+import 'package:water_tracker/custom_theme.dart';
+import 'package:water_tracker/presentation/widgets/custom_button.dart';
+import 'package:water_tracker/presentation/widgets/greeting_widget..dart';
+
+class GreetingLayout extends StatelessWidget {
+  const GreetingLayout({Key? key}) : super(key: key);
+
+  static const paddingHorizontal = EdgeInsets.symmetric(horizontal: 28.0);
+  static const spaces = SizedBox(height: 24);
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      body: Padding(
+        padding: paddingHorizontal,
+        child: Column(
+          children: [
+            const Spacer(),
+            const GreetingWidget(),
+            const Spacer(),
```

Comment:
Not sure about spacer, is it displaying correct?

Reply:
Mby u want to call spaces fields?

Reply:
It looks great, I want it